portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -123,9 +134,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       finalResponse = "Got no comment.";
       response.getWriter().println("<h3>" + finalResponse + "</h3>");
     } else {
+      System.out.println(languageCode);
```

Comment:
Just a general comment. A console print statement is critical to catching bug, so the context is very important, especially when you work on a codebase with hundreds of thousands lines of code (or even more). E.g., consider
  System.out.println("DataServlet/doPost: language code = " + languageCode);

This is just an example to illustrate the point, you could give more context in your preferred way.

Reply:
oh! I forgot to take that out. I didn't mean to include that.
But thanks